test/data/extensions/declarative_net_request/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Declarative net request",
+    "version": "0.1",
+    "manifest_version": 3,
+    "background": {},
+    "permissions": ["webRequest", "declarativeNetRequest", "declarativeNetRequestFeedback"],
```

Comment:
minor: do we need `webRequest` here ?

Reply:
removed webRequest and declarativeNetRequestFeedback